<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Touhou General Music Extractor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_fmt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_fmt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.fmt path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.dat path:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fmt">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dat">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>..</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_out">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>..</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionStart_extracting_files"/>
    <addaction name="actionExit_program"/>
   </widget>
   <widget class="QMenu" name="menuAbout_2">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_ThGME"/>
   </widget>
   <addaction name="menuAbout"/>
   <addaction name="menuAbout_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit_program">
   <property name="text">
    <string>Exit program</string>
   </property>
  </action>
  <action name="actionStart_extracting_files">
   <property name="text">
    <string>Start extracting files</string>
   </property>
  </action>
  <action name="actionAbout_ThGME">
   <property name="text">
    <string>About ThGME</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
